css/style.css

```diff
@@ -558,6 +516,10 @@ ul.works-flex-item.titles-container {
   border-radius: 10px;
 }
```

Comment:
- [ ] Your popup model work properly in mobile version but in desktop version, it doesn't appear kindly check it.